### PR TITLE
add terraform azure API management

### PR DESCRIPTION
**Pull request by usamaahmadkhan**

### CODE REVIEW

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater-lab/gac-apache-camel-integration:0.0.0-PR-7-1-SNAPSHOT`

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater-lab/gac-apache-camel-integration:0.0.0-PR-7-2-SNAPSHOT`

File: terraform/terraform.tfstate.backup [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      "type": "azurerm_api_management_api",
+      "name": "gac_camel_integration",
+      "provider": "provider.azurerm",
+      "instances": [
+        {
+          "schema_version": 0,
+          "attributes": {
+            "api_management_name": "azure-api-man",
+            "description": "",
+            "display_name": "Pliro",
```
Comment (alitariq95) [inline terraform/terraform.tfstate.backup:104]: we need to change display name

File: terraform/terraform.tfstate.backup [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+      "type": "azurerm_api_management_api_operation",
+      "name": "op-fetch",
+      "provider": "provider.azurerm",
+      "instances": [
+        {
+          "schema_version": 0,
+          "attributes": {
+            "api_management_name": "azure-api-man",
+            "api_name": "gac-camel-integration",
+            "description": "Fetch Doctors",
```
Comment (alitariq95) [inline terraform/terraform.tfstate.backup:147]: Please change description

File: terraform/terraform.tfstate.backup [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+                "header": [],
+                "query_parameter": [],
+                "representation": []
+              }
+            ],
+            "resource_group_name": "api_management_service",
+            "response": [],
+            "template_parameter": [],
+            "timeouts": null,
+            "url_template": "/api/v1/doctor"
```
Comment (alitariq95) [inline terraform/terraform.tfstate.backup:164]: We dont this anymore

**Pull request merged**